src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToStartHandler.java

```diff
@@ -3,20 +3,31 @@
 import static uk.gov.hmcts.reform.sscs.ccd.callback.DocumentType.DRAFT_ADJOURNMENT_NOTICE;
 
 import java.util.Objects;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
 import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
 import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicList;
 import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.sscs.service.PreviewDocumentService;
 import uk.gov.hmcts.reform.sscs.service.adjourncase.AdjournCaseService;
+import uk.gov.hmcts.reform.sscs.util.DynamicListLangauageUtil;
 
 @Component
 @Slf4j
 public class AdjournCaseAboutToStartHandler implements PreSubmitCallbackHandler<SscsCaseData> {
 
+    private final DynamicListLangauageUtil utils;
+
+    @Autowired
+    public AdjournCaseAboutToStartHandler(DynamicListLangauageUtil utils) {
+        this.utils = utils;
+    }
```

Comment:
You shouldn't use @Autowired, do
```suggestion
@Component
@Slf4j
@RequiredArgsConstructor
public class AdjournCaseAboutToStartHandler implements PreSubmitCallbackHandler<SscsCaseData> {

    private final DynamicListLangauageUtil utils;
```